VDF.GUI/ViewModels/MainWindowVM.cs

```diff
@@ -151,6 +152,33 @@ public sealed class CheckedGroupsComparer : System.Collections.IComparer {
 			get => _DuplicatesSelectedCounter;
 			set => this.RaiseAndSetIfChanged(ref _DuplicatesSelectedCounter, value);
 		}
+		public bool MultiOpenSupported {
+			get => !String.IsNullOrEmpty(SettingsFile.Instance.CustomCommands.OpenMultiple);
+		}
+		public bool MultiOpenInFolderSupported {
+			get => !String.IsNullOrEmpty(SettingsFile.Instance.CustomCommands.OpenMultipleInFolder);
+		}
+		static List<string> _CustomCommandList = typeof(SettingsFile.CustomActionCommands).GetProperties(BindingFlags.Public|BindingFlags.Instance).Select(p => p.Name).ToList();
+		public List<string> CustomCommandList {
+			get => _CustomCommandList;
+		}
+		PropertyInfo _SelectedCustomCommand = typeof(SettingsFile.CustomActionCommands).GetProperty(_CustomCommandList[0]);
+		public string SelectedCustomCommand {
+			get => _SelectedCustomCommand.Name;
+			set {
+				_SelectedCustomCommand = typeof(SettingsFile.CustomActionCommands).GetProperty(value);
+				this.RaisePropertyChanged(nameof(SelectedCustomCommandValue));
+			}
+		}
```

Comment:
Avalonia is currently doing some work to make trimming and AOT possible which I would like to use then. But using Reflection is not possible in AOT.